.github/workflows/validate.yml

```diff
@@ -95,6 +95,8 @@ jobs:
         with:
           go-version: "1.22"
           cache: false
+      - name: Create version.txt
```

Comment:
Why do we do this on lint and test? Just curious

Reply:
Because the current implementation requires a version.txt to be present in internal/version. It is embedded, and will give a compiler error if not present.